tests/publiccloud/instance_overview.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: This is just bunch of random commands overviewing the public cloud instance
+# We just register the system, install random package, see the system and network configuration
+# This test module will fail at the end to prove that the test run will continue without rollback
+#
+# Maintainer: Pavel Dostal <pdostal@suse.cz>
+
+use base 'consoletest';
+use registration;
+use warnings;
+use testapi;
+use strict;
+use utils;
+
+sub run {
+    my ($self, $args) = @_;
+    select_console 'root-console';
+
+    assert_script_run("hostname -f");
+    assert_script_run("uname -a");
+
+    assert_script_run("cat /etc/os-release");
+
+    assert_script_run("ps aux | nl");
+
+    register_product();
```

Comment:
this might behave differently based on type of image you use , so bad idea use it unconditionally

Reply:
How do you mean that? I just want to install `traceroute`

Reply:
Some of images available in Public Cloud expecting something like this - they called BYOS - there you getting unregistered image which suppose to register . But others will give you image which already registered via Public Cloud Provider so you don't need to register it and can just use it. Not sure how this type of images will react on attempt to register them via SCC ...

Reply:
Now I see, well, I would wait with this. Currently I use only one image and nobody decided yet for which images this will be used. The goal for this pull request is to enable the interactive ssh console in the public cloud environment. After that I will be dealing with individual tests and scenarios.